syntax = "proto2";

package proto.hashtable;

message SampledHashTable {
  required uint64 num_tables = 1;
  required uint64 reservoir_size = 2;
  required uint64 range = 3;

  repeated uint32 data = 4;
  repeated uint32 counters = 5;
  repeated uint32 gen_rand = 6;
}